### ISSUE TITLE
SQL Function(s) in class query

### ISSUE THREAD

**Issue by v23ent** [intersystems/language-server#46]
Not sure if the problem is wider than my case, but I got a "Syntax error" in a fine working SQL:

<img width="1192" alt="Снимок экрана 2020-10-27 в 18 39 32" src="https://user-images.githubusercontent.com/12609489/97325833-71dd2480-1884-11eb-83c6-58438c63ad6f.png">

**Comment by isc-bsaviano** [comment]
@v23ent Can you please paste the full text of your class query here? I will see if I can reproduce the error

**Comment by v23ent** [comment]
@isc-bsaviano  

```
/// Get the stock value at an eairliest snapshot for the date
Query dailyStockFirst(productArticleId As %String, storageId As %String, dateString As %String) As %SQLQuery
{
    SELECT TOP 1 *
    FROM isc_ecom_model.StorageStock
    WHERE productArticle = :productArticleId
        AND storage = :storageId
        AND DATE(snapshotAt) = :dateString
    ORDER BY snapshotAt
}
```

**Comment by isc-bsaviano** [comment]
@v23ent I copied your query into a class in VSCode and Studio and got the same syntax error in both places. The syntax looks fine to me so I'm going to pass this along to the team that maintains the parsers. Thanks for reporting this.

**Comment by isc-bsaviano** [comment]
@v23ent Fixed in version 1.0.4
